proto/public/alertmanager/v3/status.proto

```
@@ -0,0 +1,93 @@
+}
+
+// AlertmanagerConfig carries the currently loaded configuration.
+message AlertmanagerConfig {
+  // Original YAML configuration as a single string.
+  string original = 1;
+}
+
+// ClusterStatus describes the local node's view of the gossip cluster.
+message ClusterStatus {
+  enum State {
+    CLUSTER_STATE_UNSPECIFIED = 0;
```

Buf (1.70.0)

[error] Enum value name "CLUSTER_STATE_UNSPECIFIED" should be prefixed with "STATE_".

(ENUM_VALUE_PREFIX)